acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -7,7 +7,8 @@
 
 test_name "Install Packages"
 
-dev_builds_url  = ENV['DEV_BUILDS_URL'] || 'http://builds.delivery.puppetlabs.net'
+dev_builds_url     = ENV['DEV_BUILDS_URL'] || 'http://builds.delivery.puppetlabs.net'
+nightly_builds_url = ENV['NIGHTLY_BUILDS_URL'] || 'http://nightlies.puppet.com'
```

Comment:
@joshcooper Is this what you had in mind? Plus setting `NIGHTLY_BUILDS_URL` to ravi in your pipelines?